test/node-api/test_uv_threadpool_size/node-options.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const common = require('../../common');
+const assert = require('assert');
+const path = require('path');
+const { spawnSync } = require('child_process');
+
+if (process.config.variables.node_without_node_options) {
+  common.skip('missing NODE_OPTIONS support');
+}
+
+const uvThreadPoolPath = '../../fixtures/dotenv/uv-threadpool.env';
+
+// Should update UV_THREADPOOL_SIZE
+const filePath = path.join(__dirname, `./build/${common.buildType}/test_uv_threadpool_size`);
+const code = `
+   const { test } = require('${filePath}');
```

Comment:
This line is giving Windows issues because of all the `\n` and `\t` we have from certain parts of the path, eg. `\node\test\node-api\test_uv_threadpool_size`.

For example, this is my filePath `'E:\work\node\test\node-api\test_uv_threadpool_size\build\Release\test_uv_threadpool_size`, but if I log it from within the child process, I get this:
```
E:work
ode     est
ode-api est_uv_threadpool_sizuildRelease        est_uv_threadpool_size
```

By changing it to the following code I was able to get past loading the module, but then calling `test(size)` never finishes.
``` javascript
let filePath = path.join(__dirname, `./build/${common.buildType}/test_uv_threadpool_size`);
if (common.isWindows) {
  filePath = filePath.replaceAll('\\', '\\\\');
}
```